docs/content/latest/deploy/manual-deployment/system-config.md

```diff
@@ -166,3 +161,49 @@ In `/etc/systemd/user.conf` and `/etc/systemd/system.conf`, add at the end of fi
 
 Something similar may be needed for other distributions.
 {{< /note >}}
+
+
+## transparent hugepages
+
+Transparent hugepages should be enabled for optimal performance. By default, they are enabled.
+
+You can check with the following command:
+
+
+```sh
+$ cat /sys/kernel/mm/transparent_hugepage/enabled
+[always] madvise never
+```
+
+It is generally not necessary to adjust the kernel command line if the above value is correct.
+
+However, if the value is set to "madvise" or "never", you should modify your kernel command line to set transparent hugepages to "always".
+
+
+You should consult your operating system docs to determine the best way to modify a kernel command line argument for your operating system, but on RHEL or Centos 7 or 8, using grub2, the following steps are one solution:
+
+
+Addend "transparent_hugepage=always" to `GRUB_CMDLINE_LINUX` in `/etc/default/grub`
```

Comment:
Did you mean Append (instead of Addend).

Reply:
Yes thank you.